std_msgs/Float64MultiArray matrix
std_msgs/Float64MultiArray distortion
int64 hFOV
int64 horizontalPixels
int64 verticalPixels